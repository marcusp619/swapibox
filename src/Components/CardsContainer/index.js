import React from 'react';
import Card from '../Card/';
import './CardsContainer.css';

const CardsContainer = ({ activeCategory, people }) => {
  if(!activeCategory) {
    return (
      <section className="cards-container">   
        <h1>Please Select a Category</h1>
      </section>
    );
  } else if(activeCategory === 'People') {
    const cards = people.map((person, i) => <Card {...person} key={Date.now() + i} />);
    
    return (
      <section className="cards-container">   
        { cards }
      </section>
    );  
  } else {
    return (
      <section className="cards-container">   
        <h1>Please Select a Category</h1>
      </section>  
    )
  }
};

export default CardsContainer;
