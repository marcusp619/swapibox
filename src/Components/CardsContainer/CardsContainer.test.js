import React from 'react';
import ReactDOM from 'react-dom';
import CardsContainer from './';
import { shallow } from 'enzyme';

let wrapper

describe('App', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<CardsContainer />);
  });
})
  
beforeEach(() => {
  wrapper = shallow(<CardsContainer />);
});

it('should match snapshot', () => {
  expect(wrapper).toMatchSnapshot();
});

