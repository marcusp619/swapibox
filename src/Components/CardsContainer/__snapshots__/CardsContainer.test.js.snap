// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<section
  className="cards-container"
>
  <h1>
    Please Select a Category
  </h1>
</section>
`;
